src/main/java/oripa/geom/GeomUtil.java

```diff
@@ -26,36 +29,22 @@
 
 import javax.vecmath.Vector2d;
 
-public class GeomUtil {
-	// private static final Logger logger =
-	// LoggerFactory.getLogger(GeomUtil.class);
+import oripa.value.CalculationResource;
 
-	public final static double EPS = 1.0e-6;
+public class GeomUtil {
+	public final static double EPS = CalculationResource.POINT_EPS;
```

Comment:
I'm not sure this change is good or not. `CalculationResource.POINT_EPS` is `1e-5`, which is larger than the old value of this `EPS`.

Reply:
The bigger epsilon would only affect grid sizes around 400/1e-5=4e7 but should indeed be held as small as possible.
The global `CalculationResource.POINT_EPS` value could be updated, though I am not sure if that would brake anything.
Will try and at some point.